app/main.js

```diff
@@ -26,3 +26,5 @@ new Vue({
   render: h => h(App),
   i18n: SwiftDocCRenderi18n(),
 }).$mount('#app');
+
+document.documentElement.classList.remove('no-js');
```

Comment:
Can we actually move this slightly to be before the `$mount` call above to better guarantee that the vue app never sees this class?